Model/ResourceModel/Sources/Collection.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Ampersand\DisableStockReservation\Model\ResourceModel\Sources;
+
+use Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection;
+
+class Collection extends AbstractCollection
+{
+    /**
+     * Define resource model
+     *
+     * @return void
+     */
+    protected function _construct()
+    {
+        $this->_init(
+            'Ampersand\DisableStockReservation\Model\Sources',
```

Comment:
So we can get the classname directly from a class by doing something like `Ampersand\DisableStockReservation\Model\Sources::class`... This has the added benefit of the functionality instantly failing if the class doesn't exist, rather than relying on `_init` to validate